cli/command/system/version.go

```diff
@@ -195,11 +197,14 @@ func runVersion(dockerCli command.Cli, opts *versionOptions) error {
 			})
 		}
 	}
-
-	if err2 := tmpl.Execute(dockerCli.Out(), vd); err2 != nil && err == nil {
+	t := tabwriter.NewWriter(dockerCli.Out(), 15, 1, 1, ' ', 0)
+	buffer := bytes.NewBufferString("")
+	if err2 := tmpl.Execute(buffer, vd); err2 != nil && err == nil {
 		err = err2
 	}
-	dockerCli.Out().Write([]byte{'\n'})
+	buffer.WriteTo(t)
```

Comment:
Why do you need a buffer here? Can't you use `t` as the first arg to `tmpl.Execute` ?

Reply:
Oh! was debugging, looks like I don't need it here anymore no.